### PR TITLE
OBPIH-3414 Create order based combined shipment

### PR DESCRIPTION
**Pull request by DominikaHatala**

### CODE REVIEW

File: grails-app/views/order/_summary.gsp [hunk 1]
```diff
@@ -163,7 +163,7 @@
                                 <img src="${resource(dir: 'images/icons/silk', file: 'arrow_undo.png')}" />&nbsp;
                                 ${warehouse.message(code: 'default.button.rollback.label')}
                             </g:link>
-                            <g:link controller="stockMovement" action="createCombinedShipments" params="[direction:'INBOUND', orderId: orderInstance?.id]" class="button"
+                            <g:link controller="combineShipmentApi" action="shipOrder" params="[direction:'INBOUND', orderId: orderInstance?.id]" class="button"
```
Comment (jmiranda) [inline grails-app/views/order/_summary.gsp:166]: This isn't an API call. Why is it on the API controller?

File: src/groovy/org/pih/warehouse/api/StockMovementItem.groovy [hunk 2]
```diff
@@ -210,8 +210,10 @@ class StockMovementItem {
                 productCode: orderItem?.product?.productCode,
                 product: orderItem?.product,
                 inventoryItem: orderItem?.inventoryItem,
-                quantityRequested: orderItem.quantityRemaining,
-                recipient: orderItem.requestedBy
+                quantityRequested: orderItem.getQuantityRemainingToShip() * orderItem.quantityPerUom,
```
Comment (jmiranda) [inline src/groovy/org/pih/warehouse/api/StockMovementItem.groovy:213]: Change 
```
orderItem.getQuantityRemainingToShip()
``` 
to 
```
orderItem.quantityRemainingToShip
```

File: grails-app/views/order/_summary.gsp (hunk 1 above)
Comment (jmiranda) [reply grails-app/views/order/_summary.gsp:166]: Why doesn't the createCombinedShipments action work here?

**Comment by DominikaHatala** [comment]
@jmiranda @awalkowiak I have updated PR

**Review by awalkowiak — APPROVED** [review]

**Pull request merged**